src/tests/commonTests.ts

```diff
@@ -213,14 +213,14 @@ export async function focusable(componentTagOrHTML: TagOrHTML, options?: Focusab
   const tag = getTag(componentTagOrHTML);
   const element = await page.find(tag);
   const focusTargetSelector = options?.focusTargetSelector || tag;
-
+  await element.callMethod("componentOnReady");
```

Comment:
Is this needed? `page.waitForChanges()` [already does this](https://github.com/ionic-team/stencil/blob/16b8ea4dabb22024872a38bc58ba1dcf1c7cc25b/src/testing/puppeteer/puppeteer-page.ts#L337).